### PR TITLE
Fixed Consistent Log Levelling

### PR DESCRIPTION
**Pull request by PalmPalm7**
Edit: Have finished consistent  log leveling issue.

Made changes to exisisting codebase with structured logs from format logs. [Github issue ref](https://github.com/sustainable-computing-io/kepler/issues/1469).

Added logging kepler with klog section on dev/README.md, [see here](https://github.com/sustainable-computing-io/kepler/blob/main/doc/dev/README.md).

### CODE REVIEW

File: pkg/collector/stats/utils.go [hunk 1]
```diff
@@ -104,7 +110,7 @@ func GetNodeName() string {
 func getCPUArch() string {
 	arch, err := getCPUArchitecture()
 	if err == nil {
-		klog.V(3).Infof("Current CPU architecture: %s", arch)
+		klog.V(3).InfoS("Current CPU architecture:", "arch", arch)
```
Comment (dave-tucker) [inline pkg/collector/stats/utils.go:113]:
```suggestion
		klog.V(3).InfoS("Discovered CPU architecture", "arch", arch)
```

File: pkg/config/config.go [hunk 2]
```diff
@@ -146,14 +146,15 @@ func GetDefaultPowerModelURL(modelOutputType, energySource string) string {
...
-		klog.V(5).Infof("EXPOSE_IRQ_COUNTER_METRICS: %t", ExposeIRQCounterMetrics)
-		klog.V(5).Infof("EXPOSE_ESTIMATED_IDLE_POWER_METRICS: %t. This only impacts when the power is estimated using pre-prained models. Estimated idle power is meaningful only when Kepler is running on bare-metal or with a single virtual machine (VM) on the node.", ExposeIdlePowerMetrics)
-		klog.V(5).Infof("EXPERIMENTAL_BPF_SAMPLE_RATE: %d", BPFSampleRate)
+		klog.V(5).InfoS("Configuration value", "ENABLE_EBPF_CGROUPID", EnabledEBPFCgroupID)
+		klog.V(5).InfoS("Configuration value", "ENABLE_GPU", EnabledGPU)
+		klog.V(5).InfoS("Configuration value", "ENABLE_QAT", EnabledQAT)
+		klog.V(5).InfoS("Configuration value", "ENABLE_PROCESS_METRICS", EnableProcessStats)
+		klog.V(5).InfoS("Configuration value", "EXPOSE_HW_COUNTER_METRICS", ExposeHardwareCounterMetrics)
+		klog.V(5).InfoS("Configuration value", "EXPOSE_CGROUP_METRICS", ExposeCgroupMetrics)
+		klog.V(5).InfoS("Configuration value", "EXPOSE_IRQ_COUNTER_METRICS", ExposeIRQCounterMetrics)
```
Comment (dave-tucker) [inline pkg/config/config.go:155]: You can make this all one line:
```suggestion
		klog.V(5).InfoS("Configuration values", "ENABLE_EBPF_CGROUPID", EnabledEBPFCgroupID, "ENABLE_GPU", EnabledGPU, "ENABLE_QAT", EnabledQAT,"ENABLE_PROCESS_METRICS", EnableProcessStats, "EXPOSE_HW_COUNTER_METRICS", ExposeHardwareCounterMetrics, "EXPOSE_CGROUP_METRICS", ExposeCgroupMetrics, "EXPOSE_IRQ_COUNTER_METRICS", ExposeIRQCounterMetrics)
```

**Comment by dave-tucker** [comment]
@PalmPalm7 when you've fixed the issues with the rebase, you might want to run `make format` to ensure all your Go code is properly formatted, and `make golint` also to catch any issues.

**Comment by PalmPalm7** [comment]
> @PalmPalm7 when you've fixed the issues with the rebase, you might want to run `make format` to ensure all your Go code is properly formatted, and `make golint` also to catch any issues.

Thank you for the Review Dave, I'll make changes to ensure they are properly formatted.

**Pull request closed without merging**